components/02-form-elements/unit/unit.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="field">
+  {{> @label}}
+  <div class="input-type input-type--unit">
+    <input data-qa="input-number" id="{{id}}" class="input input--number input-type__input" name="{{id}}" value="" pattern="[0-9]*">
+    <span class="input-type__type" id="{{id}}-type">{{unit}}</span>
```

Comment:
I would like to amend `<span class="input-type__type" ...` to an `abbr` element. 
`<abbr title="{{label}}" class="input-type__type" id="{{id}}-type">{{unit}}</abbr>`

Reply:
The `<abbr>` inside an input-type will require the `text-decoration` css attribute set to none.